x-govuk/components/example/template.njk

```diff
@@ -0,0 +1,23 @@
+<div class="x-govuk-example" data-module="x-govuk-example">
+  <p class="x-govuk-example__toolbar">
+    <a class="x-govuk-example__new-window" href="{{ params.previewPath }}" target="_blank">Open this example in a new tab</a>
+  </p>
+  <div class="x-govuk-example__frame-wrapper">
+    <iframe class="x-govuk-example__frame" data-module="x-govuk-example-frame" src="{{ params.previewPath }}"></iframe>
+  </div>
+  <div class="x-govuk-example__tabs">
+    <ul class="x-govuk-example__list">
+      {%- for codeSample in params.codeSamples -%}
+        <li class="x-govuk-example__list-item">
+          <a class="x-govuk-example__list-item-link" href="#{{ params.idPrefix }}-{{ loop.index }}" role="tab" aria-controls="{{ params.idPrefix }}-{{ loop.index }}">
+            {{ codeSample.label.text }}
+          </a>
+        </li>{%- endfor -%}
+    </ul>
+  </div>
+  {%- for codeSample in params.codeSamples -%}
```

Comment:
Should this be a little less specific; perhaps call a tab a tab? What do they use in the GOV.UK Design System docs? 

```suggestion
  {%- for tab in params.tabs -%}
```

Reply:
I wasn't sure about this either.

The Design System website code has an [example component](https://github.com/alphagov/govuk-design-system/blob/main/src/stylesheets/components/_example.scss) _and_ a [custom Tabs component](https://github.com/alphagov/govuk-design-system/blob/main/src/stylesheets/components/_tabs.scss), with one nested inside the other. That felt a bit confusing to me, not to mention that 'tabs' is already a govuk component, but perhaps it’s ok.

Could go with 'items' which is the go-to when you're not quite sure what to call it? 😂